src/lib/containers/AccountLevelBadge.tsx

```diff
@@ -0,0 +1,96 @@
+import { UserBundle } from '../utils/synapseTypes'
+import * as React from 'react'
+import { useState, useEffect } from 'react'
+import { SynapseClient, SynapseConstants } from '../utils'
+import { ReactComponent as Registered } from '../assets/icons/account-registered.svg'
+import { ReactComponent as Certified } from '../assets/icons/account-certified.svg'
+import { ReactComponent as Validated } from '../assets/icons/account-validated.svg'
+import { Error } from './Error'
+
+export type AccountLevelBadgeProps = {
+  userId: string
+}
+
+export const accountLevelRegisteredLabel:string = 'Registered'
+export const accountLevelCertifiedLabel:string = 'Certified'
+export const accountLevelVerifiedLabel:string = 'Validated'
+
+export const AccountLevelBadge: React.FunctionComponent<AccountLevelBadgeProps> = ({
+  userId,
+}: AccountLevelBadgeProps) => {
+  const [error, setError] = useState()
+  const [userBundle, setUserBundle] = useState<UserBundle | undefined>(
+    undefined,
+  )
+  const [isLoading, setIsLoading] = useState<boolean>(false)
+
+  useEffect(() => {
+    const getData = async () => {
+      try {
+        setIsLoading(true)
+
+        const certificationOrVerification =
+          SynapseConstants.USER_BUNDLE_MASK_IS_CERTIFIED |
+          SynapseConstants.USER_BUNDLE_MASK_IS_VERIFIED
+
+        const bundle:UserBundle = await SynapseClient.getUserBundle(
+          userId,
+          certificationOrVerification,
+          undefined,
+        )
+        setUserBundle(bundle)
+      
+      } catch (err) {
+        setError(err)
+      } finally {
+        setIsLoading(false)
+      }
+    }
+
+    getData()
+  }, [userId])
+  if (isLoading) {
+    return <></>
```

Comment:
Just double checking that this is the desired behavior on load

Reply:
Went back and forth on whether we should have a loading progress on this component.  Since it typically loads so fast, I thought flashing a loading bar might be more distracting in this case. 
If there was a higher level component that showed many of these in a list (for example), I think that component should have a loading screen.